portsyncd/portsyncd.cpp

```diff
@@ -134,6 +135,13 @@ int main(int argc, char **argv)
                     p.set("PortInitDone", attrs);
                     SWSS_LOG_NOTICE("PortInitDone");
 
+                    /*
+                     * Notify "System ready" feature that we are ok.
+                     */
+                    FieldValueTuple app_state("up_status", "true");
```

Comment:
There should be some code to set it "false"?
1. when orchagent starts
2. when orchagent exited gracefully
3. when orchagent receive a signal and exited
4. etc.